src/Kunstmaan/ApiBundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Kunstmaan\ApiBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * This is the class that validates and merges configuration from your app/config files.
+ *
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/configuration.html}
+ */
+class Configuration implements ConfigurationInterface
```

Comment:
But it does nothing - it is used without any specific reason. It looks like generated default code by Symfony bundle generator.